marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/jobs/job-results-lib.sjs

```diff
@@ -0,0 +1,54 @@
+/**
+ Copyright (c) 2021 MarkLogic Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+'use strict';
+
+const op = require('/MarkLogic/optic');
+
+function getJobData(start, pageLength, sortColumn, sortDirection) {
```

Comment:
"JobData" is too generic here. We know exactly what we're returning - step responses. And I prefer "find" when we're doing a query, so this should be "findStepResponses".